### ISSUE TITLE
could not load library "ImageMagic" only on a notebook?

### ISSUE THREAD

**Issue by davidbp** [JuliaImages/TestImages.jl#39]
Hello,

In a terminal `testimage("cameraman")` works fine but when I use it in a jupyter notebook I get the following message:
```
Error encountered while loading "/home/david/.julia/packages/TestImages/l3RXA/images/cameraman.tif".
Fatal error:
error compiling #load_#26: could not load library "/home/david/.julia/packages/ImageMagick/d5KBL/deps/usr/lib/libMagickWand-6.Q16.so"
/lib/x86_64-linux-gnu/libz.so.1: version `ZLIB_1.2.9' not found (required by /home/david/.julia/packages/ImageMagick/d5KBL/deps/usr/lib/libpng16.so.16)

Stacktrace:
 [1] handle_error(::ErrorException, ::FileIO.File{FileIO.DataFormat{:TIFF}}) at /home/david/.julia/packages/FileIO/e8FNK/src/error_handling.jl:80
 [2] handle_exceptions(::Array{Any,1}, ::String) at /home/david/.julia/packages/FileIO/e8FNK/src/error_handling.jl:75
 [3] #load#27(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::FileIO.File{FileIO.DataFormat{:TIFF}}) at /home/david/.julia/packages/FileIO/e8FNK/src/loadsave.jl:193
 [4] load(::FileIO.File{FileIO.DataFormat{:TIFF}}) at /home/david/.julia/packages/FileIO/e8FNK/src/loadsave.jl:172
 [5] #load#13(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::String) at /home/david/.julia/packages/FileIO/e8FNK/src/loadsave.jl:118
 [6] load at /home/david/.julia/packages/FileIO/e8FNK/src/loadsave.jl:118 [inlined]
 [7] testimage(::String) at /home/david/.julia/packages/TestImages/l3RXA/src/TestImages.jl:88
 [8] top-level scope at In[4]:1
```
It's weird since I am using  (I think I am using) the same julia version.

**Comment by timholy** [comment]
Possibly dependent on the order in which you load libz-dependent packages. See https://github.com/JuliaGraphics/Gtk.jl/pull/387 and https://github.com/JuliaIO/ImageMagick.jl/issues/142#issuecomment-433691895

**Comment by timholy** [comment]
This has been fixed if you use Julia 1.3 or higher.
